### PR TITLE
Final release notes for McClintock

### PR DESCRIPTION
**Pull request by wteouchicago**
- adds SJ's release notes
- adds age of index known issues
- expand known issues for 0 cases and strict AND

### CODE REVIEW

**Comment by wteouchicago** [comment]
Hi @siosonel / @xzhou82 -- the PR is still in draft since I have a few more changes to make to the release notes, but the SJ notes are ready for your review. I edited a few words just to make things consistent with our existing release notes/text. Also, I placed all the CSS fixes into the general pile (rather than in bullets for individual apps).

**Comment by xzhou82** [comment]
Hi Wendy, all looks good. Please add this item to known defects:
In Gene Expression Clustering, the tooltip won't display when clicking an expression data cell.

File: docs/Data_Portal/Release_Notes/Data_Portal_Release_Notes.md [hunk 1]
```diff
@@ -107,9 +127,12 @@
     * Spinners on the Download Cart and Download Associated Data buttons may be displayed longer than expected. This is a visual issue and does not affect the use of these buttons. <!--SV-2343-->
     * Using multiple browser tabs with the portal when adding or removing files from the cart may result in the cart not being updated as expected. <!--SV-2412-->
 * In the __files, cases, and annotations tables__, the case ID search field is case-sensitive. If the search does not return the expected results, try changing the input to uppercase as case IDs are most commonly uppercased.
-* __Cohorts__ filtered by mutated genes and SSMs not in those genes may unexpectedly result in 0 cases. <!--SV-2331/PEAR-1616-->
+* __Cohorts__ filtered by mutated genes and SSMs not in those genes will result in 0 cases since the mutations have to belong to those particular genes in order to match cases for the results. As a workaround, first filter the cohort by the mutated genes and export the cohort using the Export Cohort feature in the Cohort Bar. Then, reimport the cohort using the Import New Cohort feature before applying the SSM filters. <!--SV-2331/PEAR-1616-->
```
Comment (wteouchicago) [inline docs/Data_Portal/Release_Notes/Data_Portal_Release_Notes.md:130]: @wwysoc2 i could not find a good way to explain strict AND (vs loose AND)...

File: docs/Data_Portal/Release_Notes/Data_Portal_Release_Notes.md [hunk 2]
```diff
@@ -107,9 +127,12 @@
...
 * The __Slide Image Viewer__ will display a black image temporarily if a user zooms in on a slide then switches to another slide. <!--SV-2370-->
 * The annotations table in the __file summary page__ does not include the Case ID column. This column is planned to be added in a future update. <!--PEAR-1899-->
+* In __ProteinPaint__, the "Gene Expression" option is non-functional when filtering samples in a sub-track.
+* In __Gene Expression Clustering__, the tooltip is not displayed when clicking an expression data cell. 
+* The custom range inputs for the __Age at Index__ card in the __Cohort Builder__ are not behaving as expected. As a workaround, use the predefined ranges available. Alternatively, use the custom range inputs on the Days tab to query for ages in years.
```
Comment (wteouchicago) [inline docs/Data_Portal/Release_Notes/Data_Portal_Release_Notes.md:135]: not loving the phrasing here (the issue is a bit confusing to explain).

**Review by clsackett — COMMENTED** [review]
One small typo

File: docs/Data_Portal/Release_Notes/Data_Portal_Release_Notes.md [hunk 3]
```diff
@@ -50,6 +50,15 @@
 * __Annotations__:
     * Annotations tables have been added to the __project, case, and file summary pages__. <!--PEAR-1853/1839/1855-->
     * Links to the case summary page have been removed for annotations that concern a redaction of the case. <!--PEAR-1942-->
+* __ProteinPaint__:
+    * A new option to toggle lollipops pointing up or down is now available.
+* __OncoMatrix__:
+    * Advanced sorting options for power users have been aded.
```
[the remark this answers is not in the document]
Comment (wteouchicago) [reply docs/Data_Portal/Release_Notes/Data_Portal_Release_Notes.md]: fixed! thanks!

File: docs/Data_Portal/Release_Notes/Data_Portal_Release_Notes.md [hunk 4]
```diff
@@ -83,6 +92,17 @@
...
+* __OncoMatrix__:
+    * Made OncoMatrix react to divide-by term edits from the label click menu.
+    * Fixed the continuous term scale for density plots.
+* __Gene Expression Clustering__:
+    * Fixed the continuous term scale for density plots.
+* __Sequence Reads__:
+    * The tool now displays the correct number of available BAM files when a cohort filter is in use.
+* __Cohort MAF__:
+    * Added the "tumor_bam_uuid" column.
+* Limited the CSS reset to avoid conflict with embedder styles, by using scoped normalize CSS rules.
```
Comment (wwysoc2) [inline docs/Data_Portal/Release_Notes/Data_Portal_Release_Notes.md:104]: Should this be "embedded"?

File: docs/Data_Portal/Release_Notes/Data_Portal_Release_Notes.md (hunk 1 above)
Comment (wwysoc2) [reply docs/Data_Portal/Release_Notes/Data_Portal_Release_Notes.md:130]: I had to focus a bit, but I understood it on the first readthrough.

**Review by wwysoc2 — APPROVED** [review]
Approved, but I'm requesting a tiny change if I'm reading the sentence correctly (otherwise disregard).

File: docs/Data_Portal/Release_Notes/Data_Portal_Release_Notes.md (hunk 4 above)
Comment (wteouchicago) [reply docs/Data_Portal/Release_Notes/Data_Portal_Release_Notes.md:104]: fixing. should be 'embedded'.

**Pull request merged**